ansible/roles/packages/tasks/repo-debian.yaml

```diff
@@ -1,8 +1,10 @@
 ---
+  - name: clean and update apt cache
+    shell: apt-get clean && apt-get update
```

Comment:
```suggestion
    command: apt-get clean && apt-get update
```

Reply:
when compared with existing jobs, the ansible playbook run took overall 1 minute longer. 

removing cleaning cache. Only updating cache is required to fix the issue as per workaround suggested in https://github.com/ansible/ansible-modules-core/issues/2951

https://teamcity.mesosphere.io/viewLog.html?buildTypeId=ClosedSource_KonvoyImageBuilder_E2eUbuntu2004&buildId=3326865&branch_ClosedSource_KonvoyImageBuilder=%3Cdefault%3E